client/app/scripts/superdesk-desks/desks.js

```diff
@@ -499,7 +499,7 @@
                                     userDesks = desks;
                                     if (desks._items.length) {
                                         if (!this.activeDeskId || !_.find(desks._items, {_id: this.activeDeskId})) {
```

Comment:
on fetchCurrentDeskId() delete the === '' condition from if
and here change the code:
                                       f (this.activeDeskId === '') {
                                            this.activeDeskId = 'personal'
                                        } else if (this.activeDeskId === null || !_.find(desks._items, {_id: this.activeDeskId})) {
                                            this.activeDeskId = desks._items[0]._id;
                                        }